Traceback (most recent call last):
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
all_about_mda = GrabWfModel(r'wfs\mdasoln.wf1', 
                  eviews_run_lines= mda_eviews_run_lines,
                  country_trans    =  mda_trans,
                    make_fitted = True,        # make equatios for fitted values of stocastic equations 
                    do_add_factor_calc=True,   # Calculate the add factors which makes the stocastic equations match    
                    fit_start = 2000,          # Start of calculation of fittet model in baseline (to have some historic values) 
                    fit_end   = None,           # end of calc for fittted model, if None taken from mdmfsa options  
                    disable_progress =True     # Better for jupyter book 
                           ) 
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mTypeError[0m                                 Traceback (most recent call last)
Cell [1;32mIn [5], line 1[0m
[1;32m----> 1[0m all_about_mda [38;5;241m=[39m GrabWfModel([38;5;124mr[39m[38;5;124m'[39m[38;5;124mwfs[39m[38;5;124m\[39m[38;5;124mmdasoln.wf1[39m[38;5;124m'[39m, 
[0;32m      2[0m                   eviews_run_lines[38;5;241m=[39m mda_eviews_run_lines,
[0;32m      3[0m                   country_trans    [38;5;241m=[39m  mda_trans,
[0;32m      4[0m                     make_fitted [38;5;241m=[39m [38;5;28;01mTrue[39;00m,        [38;5;66;03m# make equatios for fitted values of stocastic equations [39;00m
[0;32m      5[0m                     do_add_factor_calc[38;5;241m=[39m[38;5;28;01mTrue[39;00m,   [38;5;66;03m# Calculate the add factors which makes the stocastic equations match    [39;00m
[0;32m      6[0m                     fit_start [38;5;241m=[39m [38;5;241m2000[39m,          [38;5;66;03m# Start of calculation of fittet model in baseline (to have some historic values) [39;00m
[0;32m      7[0m                     fit_end   [38;5;241m=[39m [38;5;28;01mNone[39;00m,           [38;5;66;03m# end of calc for fittted model, if None taken from mdmfsa options  [39;00m
[0;32m      8[0m                     disable_progress [38;5;241m=[39m[38;5;28;01mTrue[39;00m     [38;5;66;03m# Better for jupyter book [39;00m
[0;32m      9[0m                            )

File [1;32m<string>:17[0m, in [0;36m__init__[1;34m(self, filename, modelname, eviews_run_lines, model_all_about, start, end, country_trans, country_df_trans, make_fitted, fit_start, fit_end, do_add_factor_calc, test_frml, disable_progress)[0m

File [1;32m~\anaconda3\envs\mfdoc39\lib\site-packages\ModelFlow-1.0.8-py3.8.egg\modelgrabwf2.py:264[0m, in [0;36mGrabWfModel.__post_init__[1;34m(self)[0m
[0;32m    261[0m         [38;5;28mprint[39m([38;5;124mf[39m[38;5;124m'[39m[38;5;124mNew modelflow line:[39m[38;5;132;01m{[39;00ml[38;5;132;01m}[39;00m[38;5;124m'[39m)
[0;32m    262[0m     [38;5;28;01mraise[39;00m [38;5;167;01mException[39;00m([38;5;124m'[39m[38;5;124m@ in lines [39m[38;5;124m'[39m)
[1;32m--> 264[0m [38;5;28mself[39m[38;5;241m.[39mall_frml [38;5;241m=[39m [nz[38;5;241m.[39mnormal(l,add_add_factor[38;5;241m=[39m(typ[38;5;241m==[39m[38;5;124m'[39m[38;5;124mstoc[39m[38;5;124m'[39m),make_fitted[38;5;241m=[39m(typ[38;5;241m==[39m[38;5;124m'[39m[38;5;124mstoc[39m[38;5;124m'[39m),make_fixable [38;5;241m=[39m(typ[38;5;241m==[39m[38;5;124m'[39m[38;5;124mstoc[39m[38;5;124m'[39m),eviews[38;5;241m=[39me) 
[0;32m    265[0m                  [38;5;28;01mfor[39;00m l,typ,e [38;5;129;01min[39;00m tqdm([38;5;28mzip[39m(line,line_type,eviewsline),
[0;32m    266[0m             desc[38;5;241m=[39m[38;5;124m'[39m[38;5;124mNormalizing model[39m[38;5;124m'[39m,total[38;5;241m=[39m[38;5;28mlen[39m(line),bar_format[38;5;241m=[39mbars,disable[38;5;241m=[39m[38;5;28mself[39m[38;5;241m.[39mdisable_progress)]
[0;32m    270[0m [38;5;28mself[39m[38;5;241m.[39mall_frml_dict [38;5;241m=[39m {f[38;5;241m.[39mendo_var: f [38;5;28;01mfor[39;00m f [38;5;129;01min[39;00m [38;5;28mself[39m[38;5;241m.[39mall_frml}
[0;32m    271[0m lfname [38;5;241m=[39m [[38;5;124m"[39m[38;5;124m<Z,EXO> [39m[38;5;124m"[39m [38;5;28;01mif[39;00m typ [38;5;241m==[39m [38;5;124m'[39m[38;5;124mstoc[39m[38;5;124m'[39m [38;5;28;01melse[39;00m [38;5;124m'[39m[38;5;124m'[39m [38;5;28;01mfor[39;00m typ [38;5;129;01min[39;00m line_type ]

File [1;32m~\anaconda3\envs\mfdoc39\lib\site-packages\ModelFlow-1.0.8-py3.8.egg\modelgrabwf2.py:264[0m, in [0;36m<listcomp>[1;34m(.0)[0m
[0;32m    261[0m         [38;5;28mprint[39m([38;5;124mf[39m[38;5;124m'[39m[38;5;124mNew modelflow line:[39m[38;5;132;01m{[39;00ml[38;5;132;01m}[39;00m[38;5;124m'[39m)
[0;32m    262[0m     [38;5;28;01mraise[39;00m [38;5;167;01mException[39;00m([38;5;124m'[39m[38;5;124m@ in lines [39m[38;5;124m'[39m)
[1;32m--> 264[0m [38;5;28mself[39m[38;5;241m.[39mall_frml [38;5;241m=[39m [[43mnz[49m[38;5;241;43m.[39;49m[43mnormal[49m[43m([49m[43ml[49m[43m,[49m[43madd_add_factor[49m[38;5;241;43m=[39;49m[43m([49m[43mtyp[49m[38;5;241;43m==[39;49m[38;5;124;43m'[39;49m[38;5;124;43mstoc[39;49m[38;5;124;43m'[39;49m[43m)[49m[43m,[49m[43mmake_fitted[49m[38;5;241;43m=[39;49m[43m([49m[43mtyp[49m[38;5;241;43m==[39;49m[38;5;124;43m'[39;49m[38;5;124;43mstoc[39;49m[38;5;124;43m'[39;49m[43m)[49m[43m,[49m[43mmake_fixable[49m[43m [49m[38;5;241;43m=[39;49m[43m([49m[43mtyp[49m[38;5;241;43m==[39;49m[38;5;124;43m'[39;49m[38;5;124;43mstoc[39;49m[38;5;124;43m'[39;49m[43m)[49m[43m,[49m[43meviews[49m[38;5;241;43m=[39;49m[43me[49m[43m)[49m 
[0;32m    265[0m                  [38;5;28;01mfor[39;00m l,typ,e [38;5;129;01min[39;00m tqdm([38;5;28mzip[39m(line,line_type,eviewsline),
[0;32m    266[0m             desc[38;5;241m=[39m[38;5;124m'[39m[38;5;124mNormalizing model[39m[38;5;124m'[39m,total[38;5;241m=[39m[38;5;28mlen[39m(line),bar_format[38;5;241m=[39mbars,disable[38;5;241m=[39m[38;5;28mself[39m[38;5;241m.[39mdisable_progress)]
[0;32m    270[0m [38;5;28mself[39m[38;5;241m.[39mall_frml_dict [38;5;241m=[39m {f[38;5;241m.[39mendo_var: f [38;5;28;01mfor[39;00m f [38;5;129;01min[39;00m [38;5;28mself[39m[38;5;241m.[39mall_frml}
[0;32m    271[0m lfname [38;5;241m=[39m [[38;5;124m"[39m[38;5;124m<Z,EXO> [39m[38;5;124m"[39m [38;5;28;01mif[39;00m typ [38;5;241m==[39m [38;5;124m'[39m[38;5;124mstoc[39m[38;5;124m'[39m [38;5;28;01melse[39;00m [38;5;124m'[39m[38;5;124m'[39m [38;5;28;01mfor[39;00m typ [38;5;129;01min[39;00m line_type ]

[1;31mTypeError[0m: normal() got an unexpected keyword argument 'add_add_factor'
TypeError: normal() got an unexpected keyword argument 'add_add_factor'

